docs/index.md

```diff
@@ -225,3 +225,6 @@ in our GitHub repo that will help both beginner and more advanced users.
 Have a good example you've created and want to share?
 Let us know the details via an [issue](https://github.com/mongodb/terraform-provider-mongodbatlas/issues)
 or submit a PR of your work to add it to the `examples` directory in our [GitHub repo](https://github.com/mongodb/terraform-provider-mongodbatlas/).
+
+## Terraform MongoDB Atlas Modules
+You can now leverage our [Terraform Modules](https://registry.terraform.io/namespaces/terraform-mongodbatlas-modules) to easily get started with MongoDB Atlas and critical features like [Push-based log export](https://registry.terraform.io/modules/terraform-mongodbatlas-modules/push-based-log-export/mongodbatlas/latest), [Private Endpoints](https://registry.terraform.io/modules/terraform-mongodbatlas-modules/private-endpoint/mongodbatlas/latest), etc.
```

Comment:
I would prefer to explicitly mention all terraform modules and use a bullet list instead

Reply:
IMO: Worth the cost, since we are not adding modules that often. (Feel free to resolve)

Reply:
It's not about the cost of adding new items to the list, it's more that this is something very easy to forget about, and can easily drift away from the actual list of modules, so I'd prefer to not include that dependency.